src/rpc/cli.cpp

```diff
@@ -201,7 +209,7 @@ void cli::getline( const fc::string& prompt, fc::string& line)
          line_read = readline(prompt.c_str());
          if( line_read == nullptr )
             FC_THROW_EXCEPTION( fc::eof_exception, "" );
-         rl_bind_key( '\t', rl_complete );
+         //el_bind_key( '\t', rl_complete );
```

Comment:
Why is this commented out? Is \t the default?

Reply:
Yes, \t is the default. Removed the line.

Reply:
Please push your new commit :)

Reply:
Commit lost somewhere. Redone.